overlord/snapstate/snapstate_test.go

```diff
@@ -231,7 +235,10 @@ func verifyUpdateTasks(c *C, opts, discards int, ts *state.TaskSet, st *state.St
 	expected = append(expected,
 		"set-auto-aliases",
 		"setup-aliases",
+		"run-hook",
```

Comment:
I wonder if having helpers to get "run-hook[refresh]" here wouldn't be nicer long term